lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart

```diff
@@ -87,7 +89,44 @@ class ProfilePageViewModel extends BaseModel {
   /// * `Future<void>`: Resolves when user logout
   Future<void> logout(BuildContext context) async {
     // push custom alert dialog with the confirmation message.
-    navigationService.pushDialog(logoutDialog());
+    navigationService.pushDialog(
```

Comment:
Why is logout dialog not called as before ? What is the motto behind calling it directly here ? ViewModel should not have code for View.

Reply:
This change was done by @sumitra19jha 2 years back, due to formatting and linting it has been shown here.
The profile view model contains similar functions like this.

Reply:
If so, please open an issue for making this change. Comment the issue number in this PR's itself.

Reply:
Did you open the respective issue ?

Reply:
yes